### ISSUE TITLE
ICCU: Dipl. Titel (245) unvollständig

### ISSUE THREAD

**Issue by KlausKeil** [rism-international/tasks#594]
Dipl. Titel ist meistens unvollständig. Wenn ein : oder / enthalten ist, wird er aufgetrennt und teilweise auf Lit. Feste (657) verteilt. Dipl. Titel sollte immer vollständig übernommen sein.
Beispiele: fast jeder Titel

**Comment by HirschSt** [comment]
Bitte ein **konkretes** Beispiel nennen

**Comment by KlausKeil** [comment]
850817699 Dipl. Titel in Muscat: 2. Lauda Jerusalem 
                                    bei ICCU: 2. Lauda Jerusalem : a 3 voci / Arici
850816751 Muscat: Madonnina del mare
                   ICCU:   Madonnina del mare : melodia di Grado e Caorle
850804266 Muscat: Barabano e Ruggiero per violone
                          Lit. Fest: (sec. 17.)
                   ICCU: Barabano e Ruggiero per violone : (sec. 17.) / G. Colombi

**Comment by KlausKeil** [comment]
aus Versehen geschlossen

**Comment by HirschSt** [comment]
Zunächst bereite ich eine Liste vor, bei der die Werte von 200$a und 200$f kombiniert sind, damit wir eine Übersicht und vielleicht auch ein Verständnis dafür bekommen, was genau 200$f bedeutet.

**Comment by jenniferward** [comment]
Das Feld 657 habe ich schon zum größen Teil überarbeitet und auf Felder wie Bemerkung und Besetzungshinweis verteilt! 
Ich habe dieses Ticket vor heute nicht gesehen.

**Comment by HirschSt** [comment]
Liste s. https://raw.githubusercontent.com/rism-international/import/master/data/ICCU/input/200.csv

**Comment by KlausKeil** [comment]
Jetzt habe ich einen Titel gefunden, in dem der dipl. Titel aus Unimarc 200$a, $e und $f bestehen soll: http://id.sbn.it/bid/LO11651866
200 1 $aTrois airs italiens$ela Norma, varies pour le pian$fpar F. Hunter
in der Katalogansicht: 
Titolo | Trois airs italiens : la Norma, varies pour le pian / par F. Hunter

**Comment by jenniferward** [comment]
Ein weiteres Beispiel für einen Titel, in dem der dipl. Titel aus Unimarc 200$a, $e und $f bestehen soll:
https://muscat.rism.info/admin/sources/850804763
http://id.sbn.it/bid/MIL0873097
200 1 $aNicolaus Grenon$eAd honorem sancte trinitatis$f[trascrizione di Benvenuto Maria Disertori]

in der Katalogansicht:
Titolo | Nicolaus Grenon : Ad honorem sancte trinitatis / [trascrizione di Benvenuto Maria Disertori]

In Muscat erscheinen "Ad honorem sancte trinitatis" und auch (im oben genannten Beispiel) "la Norma, varies pour le pian" im Feld 657 . Das ist ist nicht richtig; auch diese Angaben in 500 zu verschieben wäre irreführend.

Manchmal wird die Titelangabe getrennt:
https://muscat.rism.info/admin/sources/850672317
https://opac.sbn.it/bid/MSM0071461
200 1 $aTantum Ergo | a 4$epieno | N.B Si puole eseguire a Sole voci, o con accom.to di Organo Se Si vuole
Katalogansicht: 
Titolo | Tantum Ergo \| a 4 : pieno \| N.B Si puole eseguire a Sole voci, o con accom.to di Organo Se Si vuole
In Muscat, Lit. Fest: pieno | N.B Si puole eseguire a Sole voci, o con accom.to di Organo Se Si vuole
Das gehört eindeutig zum Titel, als Bemerkung wäre es fast sinnlos.

Ich finde nur 20 Angaben in 657, die tatsächlich liturgische Feste sind.

**Comment by jenniferward** [comment]
Ich brauche Bespieletitel für alle Subfelder aus 200, damit wir beurteilen können, was drin ist.

**Comment by HirschSt** [comment]
Hier die geünschte Liste: https://raw.githubusercontent.com/rism-international/import/master/data/ICCU/input/200_all.csv. Später bräuchte ich dann noch eine genaues Mapping der einzelnen Subfields auf unsere Marc-Konfiguration

done by https://github.com/rism-international/import/commit/f4b1244dca2c495f7512853988548cd879df1d84

**Comment by jenniferward** [comment]
Danke! Ich beschäftige mich mit der Liste und melde mich spätestens in der Sitzung am Montag.

Ist die Liste vom 7. Juli (Link oben) noch relevant?

**Comment by HirschSt** [comment]
_Ist die Liste vom 7. Juli (Link oben) noch relevant?_

Nein.

**Comment by jenniferward** [comment]
Erste Gedanken:
Die Liste von SH enthält 11.762 Einträgen.
$a: immer belegt: Titel
$c: 23x belegt: Laut Unimarc: Enthaltene Titel (Collections), 850811459, aber in den ICCU-Daten nicht immer (850807157)
$d: 8x belegt: Alternativer Titel  (850808273), aber 730 wäre unsauber (850793246)
$e: ca. 4.100x belegt: Weitere Titel, z.B. Untertitel (ergänzend zum Titel)
$f: immer belegt: Autorenangabe (auch Textverfasser)
$g:  ca. 1.165x belegt: Weitere Autorenangabe

Man darf nicht blind kombinieren: 
850811459 ist a, c, f
850811110 ist a, e, c, f
850810768 ist a, f, c, e

$f, $g gehören wohl in den meisten Fällen zum diplomatischen Titel, müssen aber nicht sein (eine Komponistenangabe kann ja auch z.B. auf der letzten Seite sein). 

Sicher ist immer: 
a, f, g (wenn kein c, d, e): 850776711, 850775725
a, e, f, g (wenn kein c, d): 850801306, 850632134

Sicher ist wohl:
a, d, e, f, g (wenn kein c): 850816808, 850808273

Wenn c vorhanden ist: Felder werden manchmal durch eckigen Klammern kombiniert, aber manchmal beginnt es in c und endet in e (850810768), manchmal von c nach f (850811348)

**Comment by jenniferward** [comment]
Gibt es hier weitere Kommentare? Wie machen wir weiter? Wie ich am 9. September schrieb, ist es schwierig, eine allgemeine Aussage zu treffen. Ich würde lieber eine Tabelle mit allen Feldern aus 200, für alle Titel, überarbeiten.

**Comment by HirschSt** [comment]
Die Tabelle https://raw.githubusercontent.com/rism-international/import/master/data/ICCU/input/200_all.csv enthält alle subfields

**Comment by jenniferward** [comment]
Ja, aber nur für 11.000 Titel. Kann ich bitte alle Titel haben?

**Comment by jenniferward** [comment]
Danke schön! Das ist nicht halb so schlimm. Nur ca. 13.000 Titel sind mit mehr als $a belegt. Ich kann mir die betroffenen Titel anschauen und dann ein Mapping vorschlagen.

**Comment by jenniferward** [comment]
Ich bin mit der Analyse fast durch und kann etwas über 12.000 der 13.000 Titel sagen.
Ich schlage vor, dass alle Subfelder aus ICCU-200 im 245 publiziert werden, wie bei ICCU. ICCU benutzt offenbar die ISBD-Interpunktion, ich habe nichts dagegen, aber da kommt es darauf an, welche Elemente vorhanden sind. Zum Beispiel:
(immer Blank-Zeichen-Blank)
$a / $f
$a : $e / $f ; $g
$a : $e

Ist das in Ordnung, oder hast du einen anderen Vorschlag? Kannst du die Interpunktion aus den Daten holen, oder soll ich einen Vorschlag für jede Kombination machen?

**Comment by MFalletta** [comment]
@HirschSt Kannst du dies bitte nochmal anschauen und die Frage beantworten. Danke!
